packages/palette/src/elements/Pagination/__tests__/Pagination.test.tsx

```diff
@@ -73,10 +73,10 @@ describe("Pagination", () => {
       })
 
       const prevButton = wrapper.find("[data-testid='prev']").first()
-      expect(prevButton.find("Link").prop("href")).toEqual("")
+      expect(prevButton.find("a")).toHaveLength(0)
```

Comment:
So rather than having empty hrefs; it's better to just not be a link when "disabling"